coalahtml/_coalahtml/index.html

```diff
@@ -9,9 +9,9 @@
     <link rel='stylesheet' type='text/css'
```

Comment:
@AbdealiJK  Yeah. We can create a demo page and serve it specifically for gh-pages.. That'd work